app/assets/stylesheets/scaffolds.scss

```diff
@@ -0,0 +1,65 @@
+body {
+  background-color: #fff;
```

Comment:
Since you are already using SASS, worth naming the colors and keeping all of them together in a `_variables.scss` file